### ISSUE TITLE
No option to start download the queue immediately

### ISSUE THREAD

**Issue by chitholian** [persepolisdm/persepolis#360]
**System Details:**
* Operating System: ? Linux
* Distro(For GNU/Linux and BSD users): ? Linux Mint 18.2 
* Desktop Environment(For GNU/Linux and BSD users): ?  KDE
* Persepolis Version: ? 3.0.0
* How do you install Persepolis? (Repositories, source, ...) Source

**Feature Request**
I cannot find any option to start a queue immediately during adding some links from add links window.

**Comment by alireza-amirsamimi** [comment]
Hello
Please update to version 3.0.1
Do you can start queue with 'start queue' button?

**Comment by chitholian** [comment]
Yes, I can, but I ask for a button / checkbox so that I can start processing the queue as soon as the queue added, i.e. from the text_queue.py window.

**Comment by alireza-amirsamimi** [comment]
Thanks for your feature request. It's a good suggestion. We'll work on it :)
Keep this issue open,please.
